src/main/CommandLine.cpp

```diff
@@ -902,15 +903,48 @@ runRebuildLedgerFromBuckets(CommandLineArgs const& args)
     });
 }
 
+// Transaction replay simulation that allows several modes:
+// - transactions from the archives replayed at a sustained ops-per-ledger
+// rate
+// - real and simulated transactions replayed at a sustained ops-per-ledger rate
+// - real and simulated transactions replayed with real ledger sizes scaled by
+// the multiplier
+
+// There are a few steps involved in prepping the state before we can simulate
+// transactions without state generation overhead. Typically, the sequence of
+// commands is as follows:
+// simulate-bucketlist --simulate-bucketlist <L> --multiplier <M>
```

Comment:
this comment belongs in a readme of sorts (I was suggesting creating a "transaction-simulation.md" file)